src/store/lookup.ts

```diff
@@ -312,6 +312,10 @@ async function lookupCard(
   if (await lookupCardInStock(store, page, link)) {
     const givenUrl =
       link.cartUrl && config.store.autoAddToCart ? link.cartUrl : link.url;
+    // If autoAddToCart is disabled we should not send link.cartUrl
+    if (!config.store.autoAddToCart) {
+      link.cartUrl = undefined;
```

Comment:
Oooh, was `link.cartUrl` being set to some value even if it was false in config? I manually resolved by just doing `givenUrl = link.url` in the mean time. 

Speaking of, you might still want to set `givenUrl` right?